dependencies/graknlabs/dependencies.bzl

```diff
@@ -28,7 +28,7 @@ def graknlabs_grakn_core():
     git_repository(
         name = "graknlabs_grakn_core",
         remote = "https://github.com/graknlabs/grakn",
-        tag = "1.6.2" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
+        tag = "1.7.0" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
```

Comment:
```suggestion
        tag = "1.6.2" # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
```

Reply:
this is because `client-java 1.7.0` isn't compatible with the version of `grakn-core` that `docs` depends on (i.e. `1.6.2`)

Reply:
What do you mean by "this" when you said "this is because"?

Reply:
Ah you want me to keep docs to depend on 1.6.2?

Is it because examples for client-python and client-nodejs will fail if we move to 1.7.0?

Reply:
ahh, by `this` I mean the _suggestion_. sorry, should have been clearer.

`client-java 1.7.0` isn't compatible with `grakn-core 1.6.2`. however, we do need docs to depend on `grakn-core 1.6.2` because clients Node.js and Python are yet to have a new release that works with `grakn-core 1.7.0`, therefore, we need to rollback the `client-java` version to `1.6.2`.

Reply:
@lolski ping